synapse/lib/aha.py

```diff
@@ -237,18 +235,21 @@ async def modAhaSvcInfo(self, name, svcinfo):
 
         return await self.cell.modAhaSvcInfo(name, svcinfo)
 
-    async def getAhaSvcMirrors(self, name):
+    async def getAhaSvcMirrors(self, name, iden=None):
```

Comment:
Having a mandatory name parameter which is not used if an optional param is specified is a bit too opaque. If we need an API for accessing this directly by iden, we should just plumb an API for it. Also, the proposed change seems to bypass the requirement that the user is allowed to retrieve the service?